manual_tests/test_7_full.yml

```diff
@@ -47,7 +47,7 @@
         instances:
           - host: localhost
             port: 22
-            username: root
+            agent_username: root
```

Comment:
same as previous, for consistency, maybe `agent_datadog` (or `datadog_agent`) here?

Reply:
same as above